### DISCUSSION TITLE
Remove all Elementor notes at once

### DISCUSSION THREAD

**Discussion by janheinvanasseldonk** [elementor/elementor#20935] [Feature Request]
Hi! 

Please create an option through which an admin can remove _all notes_ at once. 

I'm requesting this feature because I don't want to deliver a website where my clients can see our teams' internal communications, and it's too much of a hassle to remove them manually one by one. Of course, I am aware it can be done through phpMyAdmin, but how great would it be to be able to remove all of them with _one click of a button_ instead of having to dive into phpMyAdmin?

**Comment by jgarcianewemage** [comment]
I used to just disable them, now in 3.21 you cannot disable notes
